### PR TITLE
Metatron Launch Project *draft* Plan for Filecoin

### PR DESCRIPTION
**Pull request by Flaxscrip**
@macterra - Please review this attached Draft Project Plan. This is what I will send to the Filecoin Foundation once we agree on the content.

### CODE REVIEW

File: Projects/Metatron_Protocol_Launch/.~lock.Metatron_Plan_v.0.1.ods# [hunk 1]
```diff
@@ -0,0 +1 @@
+,flaxscrip,flaxlap,01.10.2021 13:25,file:///home/flaxscrip/.config/libreoffice/4;
```
Comment (macterra) [inline Projects/Metatron_Protocol_Launch/.~lock.Metatron_Plan_v.0.1.ods#:1]: Probably don't want to commit this file?

**Review by macterra — CHANGES_REQUESTED** [review]

File: Projects/Metatron_Protocol_Launch/.~lock.Metatron_Plan_v.0.1.ods# (hunk 1 above)
Comment (Flaxscrip) [reply Projects/Metatron_Protocol_Launch/.~lock.Metatron_Plan_v.0.1.ods#:1]: Agreed.

File: Projects/Metatron_Protocol_Launch/README.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+1. Data Notary: Document proof-of-existence (IPFS cid & Metatron xid)
+1. Data Catalog: Document storage (IPFS & Filecoin)
+1. Data Reputation: Document history traceability (Metatron xid)
+1. Data Exchange: Document versioning rights transfer (Metatron transfer)
+
+The EDEN Protocol will become a set of instructions, customizations, and lessons learned on how to deploy the Metatron & Filecoin environments in order to participate in the data exchange community being defined by the EDA.
+
+## Web3 Supply Chain Management
+
+At the outset of the Metatron Launch and EDEN Pilot, we will have implemented and demonstrated a P2P protocol for supply chain management and product traceability. While our initial user community is built around the EDA, this Web3 Supply Chain Management and Product Traceability solution will be ready to engage with other industries (food, art, medicine, etc).
```
Comment (macterra) [inline Projects/Metatron_Protocol_Launch/README.md:30]: Maybe:
> The goals of the Metatron Launch and EDEN Pilot are to implement and demonstrate a P2P protocol for supply chain management and product traceability.

File: Projects/Metatron_Protocol_Launch/README.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+## Metatron Launch and Pilot Draft Plan
+
+The *draft* plan blow provides for 3 main outcomes:
```
Comment (macterra) [inline Projects/Metatron_Protocol_Launch/README.md:34]: typo: below

File: Projects/Metatron_Protocol_Launch/README.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+1. Basic Integration of Metatron with Filecoin
+1. Launch of Metatron Protocol and Reference Implementation
+1. Configuration and Support for EDA's Pilot use of Metatron
+
+The plan currently still misses a Python software engineer with Filecoin experience. We are actively looking at finding that resource. Please contact [flaxscrip](https://flaxscrip.github.io/flaxscrip-space/) with references.
```
Comment (macterra) [inline Projects/Metatron_Protocol_Launch/README.md:40]: plan still requires a Python software engineer

File: Projects/Metatron_Protocol_Launch/README.md [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+
+![Metatron Project Deliverables](Metatron_Project_Deliverables.png)
+
+## Project Management and Governance
+
+There is a pre-Filecoin history to the project that can be preserved with the continued use of [EDENomicon](https://nomicon.edenprotocol.io/) as an open governance and transparent project management methodology. If a "clean slate" is desired, we could fork a new Nomicon game specifically for this launch. This will be determined after discussion with the Filecoin Foundation team.
+
+Nomicon is an open project management methodology that we have used successfully in numerous other software projects. For more information, please visit the [Software Development Game](https://nomicon.edenprotocol.io/Nomicon/) page on the EDENomicon.
+
+[![The Software Development Game](What_is_Nomicon.png)](https://nomicon.edenprotocol.io/Nomicon/SDG.pdf)
```
Comment (macterra) [inline Projects/Metatron_Protocol_Launch/README.md:63]: Recommend removing this image/link. The Nomicon page linked above contains links to this pdf.

File: Projects/Metatron_Protocol_Launch/README.md [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+
+1. Basic Integration of Metatron with Filecoin
+1. Launch of Metatron Protocol and Reference Implementation
+1. Configuration and Support for EDA's Pilot use of Metatron
+
+The plan currently still misses a Python software engineer with Filecoin experience. We are actively looking at finding that resource. Please contact [flaxscrip](https://flaxscrip.github.io/flaxscrip-space/) with references.
+
+![Metatron Launch and Pilot Draft Plan](Metatron_Plan.png)
+
+The plan is divided in 5 Sprints:
```
Comment (macterra) [inline Projects/Metatron_Protocol_Launch/README.md:44]: Maybe
> The plan is divided into five 4-week sprints:

**Comment by macterra** [comment]
A more general question: What blockchain will the EDA project use to authorize their assets? Metatron currently supports BTC and TESS, but will we need to integrate another one?

**Comment by macterra** [comment]
There are some other planned changes to the Metatron node that I'm wondering whether we'll need for this project:

1. full IPFS server (we're using the bare minimum currently, needs work to upgrade security, etc)
2. reverse proxy front end (to hide backend services behind single endpoint)
3. new Metatron protocol feature to support terminated assets (end of life)

File: Projects/Metatron_Protocol_Launch/README.md (hunk 2 above)
Comment (Flaxscrip) [reply Projects/Metatron_Protocol_Launch/README.md:30]: Great suggestion. Integrated in next commit.

**Comment by Flaxscrip** [comment]
@macterra - I incorporated your recommendations and added a new "Backlog" column in the kanban board to track additional future features.

**Review by macterra — APPROVED** [review]

**Pull request merged**